components/basecomponents/Exercise/exerciseStore.ts

```diff
@@ -64,12 +63,8 @@ interface WithId {
   id: number;
 }
 
-const hasSameId = (id: WithId['id']) => (obj: WithId) => id === obj['id'];
-
 export interface Choice extends WithId {
```

Comment:
Do you need `Choice` just to add an id when the `Phrase` already has one? Or could be adjusted to have one?

Reply:
I didn't want to mix up different things together. In different cases choice could be more phrases or A, B, C like in exam.